pkg/proxy/iptables/proxier.go

```diff
@@ -1497,8 +1497,8 @@ func (proxier *Proxier) syncProxyRules(reason syncReason) {
 	proxier.iptablesLines.Write(proxier.natChains.Bytes())
 	proxier.iptablesLines.Write(proxier.natRules.Bytes())
 
-	if glog.V(4) {
```

Comment:
why? I feel like it has to be 4 at least. Otherwise, there is no enough actionable information to debug e2e tests.